src/escenas/ContandoDeNuevoManic.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path="ContandoPlanetasYEstrellas.ts"/>
+
+class ContandoDeNuevoManic extends ContandoPlanetasYEstrellas {
+	automata;
+
+	iniciar(){ // TODO: DEMASIADO SELF MODIFICATION E INTROSPECTION
+		super.iniciar();
+		this.tableros.largoFila = new Tablero(0, 210, { texto: "Largo Columna Actual" , atributoObservado: 'largoColumnaActual2'});
+
+		Trait.toObject(Observado, this.automata);
+		this.automata.largoColumnaActual2 = function(){ return this.largoColumnaActual() - 1;};
+		this.automata.registrarObservador(this.tableros.largoFila);
+		this.automata.setCasillaActualViejo = this.automata.setCasillaActual;
+		this.automata.setCasillaActual = function (c,m){
+			this.setCasillaActualViejo(c,m);
+			this.changed();
+		}
```

Comment:
Me llama la atención que el this siga funcionando dentro de esta function.
AH, porque se lo lleva del contexto..... UF boludo.
Perá, ¿por qué tocás el autómata? ¿Por qué no sirve el viejo?

Hablemos porque esto no me cierra.